classes/userGroup/relationships/UserGroupStage.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * @file classes/userGroup/relationships/UserUserGroup.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class \PKP\userGroup\UserGroup
```

Comment:
The `@class` tags are supposed to be for Doxygen to identify the class that's being documented in the block, so they should always match the classname of the class being implemented in the file. (We'll probably move to PHPDocumentor at some point, which can intuit the class name -- but until then it's probably best to stay consistent.) There are a number of classes in this PR that have `@class` tags with the wrong classname in them.

Reply:
I think I fixed it for the newly created files in `/classes/userGroup/`